examples/rbpf_plugin.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Microsoft Corporation
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+// Path: examples/rbpf_plugin.rs
+use std::io::Read;
+
+fn 
+_unwind(a: u64, _b: u64, _c: u64, _d: u64, _e: u64) -> u64
+{
+    return a;
+}
+
+/**
+ * This is a plugin for the bpf_conformance test suite.
+ * It accepts a single argument, the memory contents to pass to the VM.
+ * It reads the program from stdin.
```

Comment:
We should probably add a link to your repo for the conformance tests here? It's not obvious how to use the plug-in otherwise